src/Codeception/Lib/Interfaces/Web.php

```diff
@@ -700,18 +700,32 @@ public function uncheckOption($option);
     public function fillField($field, $value);
 
     /**
-     * Attaches a file relative to the Codeception `_data` directory to the given file upload field.
+     * Fills field with the given value.
      *
      * ``` php
      * <?php
-     * // file is stored in 'tests/_data/prices.xls'
-     * $I->attachFile('input[@type="file"]', 'prices.xls');
+     * $I->fillField('#username', 'username');
      * ?>
      * ```
      *
      * @param $field
-     * @param $filename
+     * @param $value
      */
+    
+    public function clearField($field);
+
+    /**
+     * Clears given field which isn't empty.
+     *
+     * ``` php
+     * <?php
+     * $I->clearField('#username');
+     * ?>
+     * ```
+     *
+     * @param $field
+     */
```

Comment:
Please revert changes to this file. Also docblock should go **JUST BEFORE THE METHOD** it descibes